src/docs/asciidoc/user-guide/assertj-core-migration.adoc

```diff
@@ -20,24 +20,25 @@ assertThat(actual).isEqualTo(expected);
 There are several ways to perform the conversion :
 
 * Automatically using the provided link:#assertj-migration-using-scripts[migration scripts].
+* Automatically with link:#assertj-migration-using-openrewrite[OpenRewrite].
 * Manually using the regexes described in this link:#assertj-migration-using-regexes[section].
 * With IntelliJ IDEA, using these plugins:
 ** https://plugins.jetbrains.com/plugin/10345-assertions2assertj[Assertions2Assertj] plugin.
 ** https://plugins.jetbrains.com/plugin/12195-concise-assertj-optimizing-nitpicker-cajon-[Concise AssertJ Optimizing Nitpicker (Cajon)] plugin.
 * With IntelliJ IDEA, using the Structural Search and Replace (SSR) feature.
 
-The preferred way is using the scripts as it is quicker and covers more assertions that the other ones.
+The preferred approach is to use the provided migration scripts or OpenRewrite recipes as they are quicker and cover more assertions than the other ones.
```

Comment:
Note quite sure if we're quicker given that we might need to build up a large LST, but we're definitely safer.
```suggestion
The preferred approach is to use the provided migration scripts or OpenRewrite recipes as they are type safe and cover more assertions than the other ones.
```